internal/middleware/request_logging.go

```diff
@@ -0,0 +1,42 @@
+package middleware
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+)
+
+func AddRequestLogging(logs logrus.FieldLogger) mux.MiddlewareFunc {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+			if strings.Contains(req.URL.Path, "service_bindings") {
+				logs.WithFields(logrus.Fields{
+					"method":      req.Method,
+					"url":         req.URL.String(),
+					"query":       req.URL.Query(),
+					"headers":     req.Header,
+					"remote_addr": req.RemoteAddr,
+					"host":        req.Host,
+					"user_agent":  req.UserAgent(),
+				}).Info("Request details")
+
+				if req.Body != nil {
```

Comment:
We already logged "Request details" - we can consider logging some message if the body is nil.
It can be inferred, but such explicitly put message would make analysis easier.